src/app/settings/settings.module.ts

```diff
@@ -1,10 +1,8 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { ClipboardModule, ClipboardService } from 'ngx-clipboard';
-
```

Comment:
It's a good idea to separate third party imports and local imports. This described in Angular Style Guide

Reply:
Add optimize imports sort by module